WordPress/src/main/java/org/wordpress/android/ui/sitecreation/sitename/SiteCreationSiteNameViewModel.kt

```diff
@@ -38,26 +38,23 @@ class SiteCreationSiteNameViewModel @Inject constructor(
 
     fun start() {
         if (isInitialized) return
-        // TODO: analyticsTracker.trackSiteNameViewed()
-        Log.d("siteNameWip", "Site name viewed")
+        analyticsTracker.trackSiteNameViewed()
         isInitialized = true
     }
 
     fun onSkipPressed() {
-        // TODO: analyticsTracker.trackSiteNameSkipped()
-        Log.d("siteNameWip", "Site name skipped")
+        analyticsTracker.trackSiteNameSkipped()
         _onSkipButtonPressed.call()
     }
 
     fun onBackPressed() {
-        // TODO: analyticsTracker.trackSiteNameCancelled()
-        Log.d("siteNameWip", "Site name cancelled")
+        analyticsTracker.trackSiteNameCanceled()
         _onBackButtonPressed.call()
     }
 
     fun onSiteNameEntered() {
-        // TODO: analyticsTracker.trackSiteNameEntered(siteName?) maybe we don't want to track the names here?
         uiState.value?.siteName.let {
+            analyticsTracker.trackSiteNameEntered(it)
             if (it.isNullOrBlank()) return
             Log.d("siteNameWip", "Site name entered: $it")
```

Comment:
I forgot to remove this :man_facepalming:

Reply:
Cleaned up with https://github.com/wordpress-mobile/WordPress-Android/pull/16335/commits/b2379edd2c316246046c6f49109d21f5de54083f.